Resources/config/services.yml

```diff
@@ -1,88 +1,39 @@
 services:
```

Comment:
Dunno why you removed all of this stuff. You are supposed to just add quotes for Symfony 3 support.